roll/extensions.py

```diff
@@ -125,7 +125,7 @@ def simple_server(app, port=3579, host='127.0.0.1', quiet=False):
         app.loop.close()
 
 
-def static(app, prefix='/static/', root=Path()):
+def static(app, prefix='/static/', root=Path(), fallback=''):
```

Comment:
Naming: I understand `fallback` more as "if I can't do what I'm supposed to do, I'll *fallback*".

Could it be `default` or is that more or less the same ? Maybe `index=` ?

Maybe we could see how this is handled and named in an other framework (or eg. in Nginx), where I guess this heuristic exists too ?

Reply:
`default_index` ? as in https://github.com/nginx/nginx/blob/15544440425008d5ad39a295b826665ad56fdc90/src/http/modules/ngx_http_index_module.c#L26